scripts/globals/interaction/container.lua

```diff
@@ -145,6 +145,20 @@ function Container:unsetVarBit(player, name, bitNum)
     end
 end
 
+function Container:countSetVarBits(player, name)
+    local currentValue = player:getVar(self.varPrefix .. name)
+    local count = 0
+    while currentValue > 0 do
+        if bit.band(currentValue, 1) == 1 then
+            count = count + 1
+        end
+
+        currentValue = bit.rshift(currentValue, 1)
+    end
+
+    return count
+end
```

Comment:
Pretty nifty. For passers by: This is shifting and truncating the target number until it comes back zero, which means we've hit and counted the last possible set bit - so we can stop iterating.

Reply:
I'll remove this